src/views/start.njk

```diff
@@ -4,21 +4,72 @@
 
 {% block content %}
   <h1 class="govuk-heading-xl">
-    Trap Registration Return
+    Trap Registration - meat bait returns
   </h1>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <p class="govuk-body">
-        Use this service to submit your trap registration return
+        If you have registered to use meat baits in:
       </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>a Larsen mate trap</li>
+        <li>a Larsen pod trap</li>
+      </ul>
+      <p class="govuk-body">
+        Under NatureScot's General Licenses:
+      </p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>
+          <a class="govuk-link" href="https://gateway.snh.gov.uk/go/?id=18">GL01
+           - To kill or take certain birds for the conservation of wild birds</a>
+        </li>
+        <li>
+          <a class="govuk-link" href="https://gateway.snh.gov.uk/go/?id=19">GL02
+           - To kill or take certain birds for the prevention of serious damage</a>
+        </li>
+      </ul>
+      <p class="govuk-body">
+        Please use this service to provide your annual license return.
+      </p>
+      <p class="govuk-body">
+        Providing your return takes around 5 minutes
+      </p>
+      {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+      {{ govukWarningText({
+         text: "You must provide a return if you have registered to use meat baits
+         even if you have not caught any non-target species.",
+         iconFallbackText: "Warning"
+      }) }}
       <form method="post">
         {{ govukButton({
           text: "Start now",
           classes: "govuk-!-margin-top-2 govuk-!-margin-bottom-8 naturescot-forward-button",
           isStartButton: true
         }) }}
       </form>
+      <p class="govuk-body">
+      If you need any assistance with this registration, vontact the NatureScot Liscensing Team.
```

Comment:
Should be `contact`?

Reply:
Resolved in https://github.com/Scottish-Natural-Heritage/trap-registration-return/pull/7/commits/f0334437e70fd46a2e2c5abc78b1c6645feab2b4